data/json/npcs/hells_raiders/TALK_BANDIT_LEADER.json

```diff
@@ -0,0 +1,126 @@
+[
+  {
+    "id": "TALK_HELLS_RAIDERS_BANDIT_LEADER",
+    "type": "talk_topic",
+    "dynamic_line": "I ain't seen you here before.  New?",
+    "responses": [
+      { "text": "Yes, I'm new.  Nice to meet you.", "topic": "TALK_HELLS_RAIDERS_BANDIT_LEADER_NEW" },
+      { "text": "Hate to run, but I gotta go.  See ya.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "id": "TALK_HELLS_RAIDERS_BANDIT_LEADER_NEW",
+    "type": "talk_topic",
+    "dynamic_line": "Don't expect a flowery time, then.  Life can be hard, tough, and full of blood and shit.  If you can't handle that, then you're out on your ass.  Got it?",
+    "responses": [
+      { "text": "Got it.", "topic": "TALK_HELLS_RAIDERS_BANDIT_LEADER_2" },
+      { "text": "What's with the tough introduction?", "topic": "TALK_HELLS_RAIDERS_BANDIT_LEADER_NEWRECRUIT" },
+      { "text": "I'm not ready for this.", "topic": "TALK_HELLS_RAIDERS_BANDIT_LEADER_LEAVE" }
+    ]
+  },
+  {
+    "id": "TALK_HELLS_RAIDERS_BANDIT_LEADER_2",
+    "type": "talk_topic",
+    "dynamic_line": [ "What'cha want?", "Make it quick.", "I've got paperwork to do, let's make it snappy.", "Let's get going." ],
+    "speaker_effect": { "effect": { "npc_first_topic": "TALK_HELLS_RAIDERS_BANDIT_LEADER_2" } },
+    "responses": [
+      { "text": "Anything that you need done around here?", "topic": "TALK_MISSION_LIST" },
+      { "text": "Who exactly are you guys anyway?", "topic": "TALK_HELLS_RAIDERS_BANDIT_LEADER_WHO" },
+      { "text": "What do you do these days?", "topic": "TALK_HELLS_RAIDERS_BANDIT_LEADER_JOB" },
+      { "text": "Could you loan a poor soul any supplies?", "topic": "TALK_HELLS_RAIDERS_BANDIT_NOBORROW" },
+      { "text": "Would you like to buy anything?", "topic": "TALK_HELLS_RAIDERS_BANDIT_NOTRADE" },
+      { "text": "Goodbye.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "id": "TALK_HELLS_RAIDERS_BANDIT_LEADER_WHO",
+    "type": "talk_topic",
+    "dynamic_line": "Some people call us thieves, others scum, the <swear> government calls us the Hell's Raiders.  We're just some folks doin' what we need to survive is all.",
```

Comment:
ditto.
Also, this dialogue is a duplicate of that in the last topic, surely it could be combined?